content/en/docs/tasks/extensibility/_index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Extensibility
+description: Demonstrates how to distribute and configure a Wasm module.
```

Comment:
wdyt about leaving the possibility open for more than Wasm modules here?

Maybe: "Demonstrates how to extend mesh behavior." ?

Reply:
Seems to make sense. Will change this.

Reply:
Done.